### PR TITLE
Fancy Bricks!

### PR DESCRIPTION
**Pull request by IIAmMaker**

### CODE REVIEW

File: fonts/minecraft-ten/textures.json [hunk 1]
```diff
@@ -213,7 +213,12 @@
       }
     },
       "bricks": {
-        "author": "IAmMaker"
+        "author": "IAmMaker",
+        "variants": {
+          "fancy_bricks":{
+            "name": "Fancy Bricks"
```
Comment (oliviathevampire) [inline fonts/minecraft-ten/textures.json:219]: This is not really needed as it will automatically create the name from the file

Comment (IIAmMaker) [reply fonts/minecraft-ten/textures.json:219]: ok

**Comment by ewanhowell5195** [comment]
could you switch them round so these bricks are the primary bricks and the old ones are a variant

**Comment by ewanhowell5195** [comment]
give these new ones the `bricks` id

**Comment by IIAmMaker** [comment]
> give these new ones the `bricks` id
ok

**Comment by IIAmMaker** [comment]
> give these new ones the `bricks` id

ok

**Pull request merged**